Src/BootCamp.Chapter/Shop.cs

```diff
@@ -12,12 +12,13 @@ public decimal GetMoney()
 
         public Shop()
         {
-
+            
```

Comment:
This should also set _inventory so it isn't null anymore.

Reply:
solved